RN 41511 (#4121)
SkylineCommunications/dataminer-docs: develop/api/NotifyTypes/NT_FILL_ARRAY_WITH_COLUMN.md

  <ColumnOption idx="0" pid="1001" type="retrieved" options="" />
  ```

> - The column data should always be wrapped in object arrays for the data to be correctly interpreted by the SLProtocol process. If an invalid type, like a string array, is passed, the function will fail and throw an error, which will automatically be logged in the element's log file. Prior to DataMiner 10.4.0 [CU11]/10.5.2<!--RN 41511-->, the function fails silently in such cases, without logging or throwing an error.

Reviewer: ```md
- The column data should always be wrapped in an object array or string array for the data to be correctly interpreted by the SLProtocol process. If any other type is passed, the function will fail and throw an error, which will automatically be logged in the element's log file. Prior to DataMiner 10.4.0 [CU11]/10.5.2<!--RN 41511-->, the function fails silently in cases where something else than an object array was passed, without logging or throwing an error.
````

Author: Thanks for the correction, @BramDevlaminck-Skyline. I've updated the text based on your input.

```suggestion
- The column data should always be wrapped in object or string arrays for the data to be correctly interpreted by the SLProtocol process. If a type other than an object or string array is passed, the function will fail and throw an error, which will automatically be logged in the element's log file. Prior to DataMiner 10.4.0 [CU11]/10.5.2<!--RN 41511-->, the function fails silently if anything other than an object array is passed, without logging or throwing an error.
```
